src/@commonsku/styles/icons/SkubotSpinner.tsx

```diff
@@ -100,27 +102,34 @@ const Path = styled.circle<{ color: string }>`
   stroke-dasharray: 187;
   stroke-dashoffset: 0;
   transform-origin: center;
-  animation: ${dash} 1.4s ease-in-out infinite, ${props => colors(props.color)} 5.6s ease-in-out infinite;
+  animation: ${dash} 1.4s ease-in-out infinite,
+    ${(props) => colors(props.color)} 5.6s ease-in-out infinite;
 `;
 
 export default function SkubotSpinner({
   color = teal.main,
   size = "default",
   altText = "loading",
-  skubot = true,
+  skubot = containerSizes[size] !== "icon",
```

Comment:
I think you probably mean `skubot = size !== "icon"`, but aside from that, I don't think this component will be usable with a `size` prop of `"icon"` because `size` is `TIconSize?` and `TIconSize` doesn't include `"icon"`.
I think the most straightforward fix is probably to create a new type and update the SkubotSpinner props and defaults accordingly.  e.g.
```
type TContainerSize = keyof typeof containerSizes;
export type SkubotSpinnerProps = {
  size?: TIconSize;
  containerSize?: TContainerSize;
  width?: string | number;
  height?: string | number;
  altText?: string;
  color?: string | undefined;
  skubot?: boolean
} & SharedStyleTypes;

export default function SkubotSpinner({
  color = teal.main,
  size = "default",
  containerSize = size,
  altText = "loading",
  skubot = containerSize !== "icon",
  ...props
}: SkubotSpinnerProps) {
  return (
    <Container size={containerSizes[containerSize] ? containerSizes[containerSize].width : 200}>
    ... etc.
```

Reply:
What about adding `icon` type into size as an addition? 

`export type SkubotSpinnerProps = {
  size?: TIconSize | 'icon';
  containerSize?: TContainerSize;
  width?: string | number;
  height?: string | number;
  altText?: string;
  color?: string | undefined;
  skubot?: boolean;
} & SharedStyleTypes;`

Reply:
Maybe something like:
```
export type SkubotSpinnerProps = {
 size?: TIconSize; 
 width?: number;
 altText?: string; 
 color?: string;
 skubot?: boolean;
} & SharedStyleTypes;
```
And then use
```
const containerWidth  = containerSizes[size]?.width ?? width?? containerSizes.default.width;
```
To calculate the width of the container, and then you don't need an "icon" size.